cadaver: add page (#11092)
tldr-pages/tldr: pages/common/cadaver.md

# cadaver

> > A command-line WebDAV client for Unix.

Reviewer: ```suggestion
> WebDAV client for Unix.
```

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions

> More information: <https://manned.org/cadaver>.

> - Connect to the server dav.example.com, open the root collection:

Reviewer: ```suggestion
- Connect to the server <dav.example.com>, open the root collection:
```

We can enclose the link inside `<>` to make it clickable.

> `cadaver http://dav.example.com/`

Reviewer 2: ```suggestion
`cadaver {{http://dav.example.com/}}`
```

> - Connect to the server dav.example.com use port 8022 and open the collection "/foo/bar/":

Reviewer 2: ```suggestion
- Connect to a server using a specific port and open the collection "/foo/bar/":
```

> `cadaver http://dav.example.com:8022/foo/bar/`

Reviewer 2: ```suggestion
`cadaver {{http://dav.example.com:8022/foo/bar/}}`
```

> - Connect to a server called davs.example.com use SSL:

Reviewer 2: ```suggestion
- Connect to a server using SSL:
```

> `cadaver https://davs.example.com/`

Reviewer 2: ```suggestion
`cadaver {{https://davs.example.com/}}`
```
